charts/alfresco-transform-service/templates/deployment-filestore.yaml

```diff
@@ -63,6 +67,24 @@ spec:
             - name: data
               mountPath: {{ .Values.filestore.persistence.data.mountPath }}
               subPath: {{ .Values.filestore.persistence.data.subPath }}
+            {{- with .Values.filestore.volumeMounts }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
       volumes:
       {{- include "data_volume" .Values.filestore | nindent 8 }}
+      {{- with .Values.filestore.volumes }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.filestore.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.filestore.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
duplicates the affinity definition in line 33.
Please move it to the default values and replace the deprecated `failure-domain.beta.kubernetes.io/zone` by the newer `topology.kubernetes.io/zone`